CentralPackageManagementExample/src/ClassLibraryWithVersionOverride/ClassLibraryWithVersionOverride.csproj

```diff
@@ -0,0 +1,14 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>net6.0</TargetFramework>
+    <ImplicitUsings>enable</ImplicitUsings>
+    <Nullable>enable</Nullable>
+  </PropertyGroup>
+  <ItemGroup>
+    <!--
```

Comment:
This example is confusing as it talks about transitive pinning but is really about version overrides? The text also mentions a 12.0.1 and a 13.0.1. The packages.props file contains a 1.0.2 for the Bson and this csproj is 9.0.1 and it's getting confusing to me.

Perhaps this could be reworded a bit to explain the relationships better?

Reply:
Whoops yeah that's a mistake, fixed!